libs/expo/shared/ui-components/src/lib/Textarea/Textarea.tsx

```diff
@@ -52,6 +52,7 @@ interface ITextareaProps extends TextInputProps {
   ml?: TSpacing;
   mr?: TSpacing;
   height?: number | 'auto';
+  textAlignVertical?: 'top' | 'center' | 'bottom';
```

Comment:
i don't believe you'd need this line since  ITextareaProps already extends `TextInputProps`

Reply:
Tested this and yes its redundant, thanks for the feedback. Have removed